actors/miner/tests/prove_commit.rs

```diff
@@ -13,12 +8,17 @@ use fvm_shared::{
     sector::{StoragePower, MAX_SECTOR_NUMBER},
     smooth::FilterEstimate,
 };
-use std::collections::HashMap;
-
-mod util;
 
+use fil_actor_miner::{
```

Comment:
Where do these rearrangments come from? Is this some change in cargo fmt?

Reply:
Thanks for picking this up. It comes from [my IDE's "Optimize Imports" function](https://github.com/intellij-rust/intellij-rust/issues/2259), which regroups imports into std, external crates, workspace crates, etc. Rustfmt by default doesn't care about import grouping, only alphabetical order within a group.

The [group imports](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports)  feature could be turned on, which would I think then completely specify the grouping and order (similar but slightly different to my IDE). After a one-time churn, this could reduce noise here in the future.

In the meantime I will attempt to avoid regrouping imports this way.